7/s2i/bin/assemble

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -e
+
+echo "---> Build started at `date`"
+
+echo "---> Preparing source..."
+cp -Rf /tmp/src/. .
+
+# fix source directory permissions
+fix-permissions ./
+
+################################################################################
+# Example assemble script to be placed in $srcdir/.s2i/bin/assemble
+################################################################################
+#
+# #!/bin/bash
+#
+# # Source base assemble script.
+# source /usr/libexec/s2i/assemble 
+# 
+# mkdir build
+# cd build
+# cmake .. -DCMAKE_BUILD_TYPE=Release
+# make -j`nproc`
```

Comment:
This is a nice example, what about to put this into the usage as well? Or README.md? Or both? :)

Reply:
Is this assemple script example inside of assemble script itself?
Or is it supposed to be a `customized` assemble script, which overrides the original one?
Specifically the difference between `./.s2i/..` and `./s2i/..` - note the dot.

Please forgive my ignorance.